indico/web/forms/widgets.py

```diff
@@ -190,6 +195,36 @@ def __call__(self, field, **kwargs):
                                                      search_url=self.search_url, choices=getattr(field, 'choices', []))
 
 
+class LocationWidget(JinjaWidget):
+    """Renders a collection of fields to represent location, enhanced with
+       jquery-typeahead
```

Comment:
*"enhanced with jquery-typeahead"* is a bit verbose (it'd fit perfectly in one line without that!)